apps/src/music/blockly/blocks/simple2.js

```diff
@@ -310,6 +310,34 @@ export const playSoundsTogether = {
     `,
 };
 
+export const callChorusSimple2 = {
+  definition: {
+    type: BlockTypes.CALL_CHORUS_SIMPLE2,
+    message0: 'chorus',
```

Comment:
Assuming we will be translating start sources, how do we ensure this block is created correctly (with the right label)?

Reply:
Good question.  Do we typically translate start sources?  If so, what's the mechanism for that?

Reply:
We do have this translation code but currently Music Lab is not hooked into this process, so I don't believe start sources are translated. It also looks like it may only work on Blockly XML?

That being said, I believe, we localize blockly block text client side at runtime, so for most cases I'm not sure if we need to send sources through our i18n pipeline? Unless there are some level-defined variables/fields that we don't have client strings for.